README.md

```diff
@@ -42,7 +42,11 @@ var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
 Create a For Loop that outputs the following string for each piece of fruit in the console. "I want to eat a [fruit]"
 
 ```js
-// write code here
+ for(i in foods){
+   for (j in food[i]){
+     console.log("I want to eat "+foods[i][j])
+   }
+ }
 ```
```

Comment:
:+1: Although don't forget to pluralize `food` in your inner loop. Otherwise, this'll break.